panel/src/components/Layout/Header.vue

```diff
@@ -81,6 +85,9 @@ export default {
   margin-bottom: 2rem;
   border-bottom: 1px solid var(--color-border);
 }
+.k-header[data-tabs] {
```

Comment:
Remember that we should already do `="true"`. Vue 3 will render data attributes with `false` as value still as `="false"` and your current CSS selector would still match that